src/metatensor/models/soap_bpnn/model.py

```diff
@@ -282,17 +282,20 @@ def forward(
         outputs: Dict[str, ModelOutput],
         selected_atoms: Optional[Labels] = None,
     ) -> Dict[str, TensorMap]:
-        if selected_atoms is not None:
-            raise NotImplementedError("SOAP-BPNN does not support selected atoms.")
-
+        # check outputs
         for requested_output in outputs.keys():
             if requested_output not in self.capabilities.outputs.keys():
                 raise ValueError(
                     f"Requested output {requested_output} is not within "
                     "the model's capabilities."
                 )
```

Comment:
sorry, just realized this one is already checked by `MetatensorAtomisticModel` as well.

Reply:
Ok! I'll take it out